admin_guide/install/deploy_router.adoc

```diff
@@ -139,6 +139,22 @@ $ oadm router <router_name> --replicas=<number> \
     --credentials="$KUBECONFIG" \
     --service-account=router
 ----
+
+If the command fails with below error, make sure that  $KUBECONFIG have the correct path
+and rerun the command.
+
+----
+error: router could not be created; you must specify a .kubeconfig file path
+containing credentials for connecting the router to the master with --credentials
+----
+
+The default $KUBECONFIG is present in /etc/openshift/master/openshift-router.kubeconfig.
```

Comment:
What install method were you observing this default? It's tough for OpenShift Origin (which is where this content is appearing because of the `ifdef::openshift-origin` above) since the location could be different based on the install method (versus OpenShift Enterprise where we know the default configs will be in `/etc/openshift/` somewhere). For example, from an install using Vagrant:

```
# echo $KUBECONFIG
/openshift.local.config/master/admin.kubeconfig
```

Note that it's the overall `admin.kubeconfig` there, versus the router-specific `openshift-router.kubeconfig`.

I agree that we should add some help in this spot in case `KUBECONFIG` is not set (it was left vague at the time based on the above uncertainties). LMK what you're thinking and we can figure something out.